frontend/src/metabase/admin/upsells/UpsellHosting.tsx

```diff
@@ -0,0 +1,53 @@
+import { t, jt } from "ttag";
+
+const RocketGlobeIllustrationSrc = "app/assets/img/rocket-globe.svg";
+import { useSelector } from "metabase/lib/redux";
+import { getIsHosted } from "metabase/setup/selectors";
+
+import { UpsellCard } from "./components";
+
+export const UpsellHosting = ({ source }: { source: string }) => {
+  const isHosted = useSelector(getIsHosted);
+
+  if (isHosted) {
+    return null;
+  }
+
+  return (
+    <UpsellCard
+      title={t`Minimize maintenance`}
+      campaign="hosting"
+      buttonText={t`Learn more`}
+      buttonLink="https://www.metabase.com/cloud"
+      illustrationSrc={RocketGlobeIllustrationSrc}
+      source={source}
+    >
+      {jt`${(
+        <strong>{t`Migrate to Metabase cloud`}</strong>
+      )} for fast, reliable, and secure deployment.`}
+    </UpsellCard>
+  );
+};
+
+export const UpsellHostingUpdates = ({ source }: { source: string }) => {
+  const isHosted = useSelector(getIsHosted);
+
+  if (isHosted) {
+    return null;
+  }
+
+  return (
+    <UpsellCard
+      title={t`Get automatic updates`}
+      campaign="hosting"
+      buttonText={t`Learn more`}
+      buttonLink="https://www.metabase.com/cloud"
+      illustrationSrc={RocketGlobeIllustrationSrc}
+      source={source}
+    >
+      {jt`${(
+        <strong>{t`Migrate to Metabase cloud`}</strong>
```

Comment:
```suggestion
        <strong>{t`Migrate to Metabase Cloud`}</strong>
```

Reply:
@luizarakaki ?

Reply:
(I had a similar thought, but all the designs had that casing)